services/workflows-service/src/business/business.service.ts

```diff
@@ -205,3 +311,14 @@ export class BusinessService {
     }
   }
 }
+
+const retry = async (fn: () => Promise<unknown>) => {
+  const { default: pRetry } = await import('p-retry');
```

Comment:
why dynamic import?

Reply:
`p-retry` supports only ESM, and we use CJS. Dynamic import is a wrokaround to make it work in our project.